docker-compose.dev.yml

```diff
@@ -14,6 +14,10 @@ services:
       args:
         USER_UID: ${USER_UID}
         SDK_VERSION: ${SDK_VERSION:-latest}
+        NEWRELIC_ENABLED: ${NEWRELIC_ENABLED}
+        NEWRELIC_LICENSE: ${NEWRELIC_LICENSE}
```

Comment:
I guess it should be moved to env variables (NEWRELIC_LICENSE, NEWRELIC_APPNAME, NEWRELIC_LOGLEVEL)

Reply:
thanks, fixed